src/Language/GraphQL/Draft/Printer.hs

```diff
@@ -236,20 +236,27 @@ variableP v = charP '$' <> printP v
 
 value :: (Print var, Printer a) => Value var -> a
 value = \case
-  VVariable v -> variableP v
-  VInt i      -> intP i
-  VFloat d    -> doubleP d
-  VString s   -> stringValue s
-  VBoolean b  -> fromBool b
-  VNull       -> "null"
-  VList xs    -> listValue xs
-  VObject o   -> objectValue o
-  VEnum ev    -> nameP $ unEnumValue ev
+  VVariable v    -> variableP v
+  VInt i         -> intP i
+  VFloat d       -> doubleP d
+  VString t s    -> dispatchStringPrinter t s
+  VBoolean b     -> fromBool b
+  VNull          -> "null"
+  VList xs       -> listValue xs
+  VObject o      -> objectValue o
+  VEnum ev       -> nameP $ unEnumValue ev
+
+dispatchStringPrinter t s = case t of
+  StringCharacter -> stringValue s
+  BlockStringCharacter -> blockStringValue s
 
 -- | We use Aeson to decode string values, and therefore use Aeson to encode them back.
 stringValue :: Printer a => Text -> a
 stringValue s = textP $ LT.toStrict $ LTE.decodeUtf8 $ J.encode s
 
+blockStringValue :: Printer a => Text -> a
+blockStringValue t = textP "\"\"\"\n" <> textP t <> textP "\n\"\"\""
+
```

Comment:
This fails in the same way that `stringValue` used to before it was fixed in #44.

Reply:
That means we cannot use the same constructor for both normal strings and block strings, or Am I missing something here?

Reply:
It's not failing anymore with the new parser.